deploy/configs/applications/modules.yaml

```
@@ -83,6 +83,7 @@ modules:
         '^python@3:': '/python3'
         '+mpi': '/parallel'
         '~mpi': '/serial'
+        '+common': '+common'  # neurodamus-core
```

Are we deploying two modules of neurodamus-core? are both as modules required?

By the way to avoid error as serial/parallel, it should be : 
```suggestion
        '^neurodamus-core+common': '+common'  # neurodamus-core
```

(I don't expect packages with variant +common but still...)

---

> Are we deploying two modules of neurodamus-core? are both as modules required?

Yep if we still want to support the case Werner had.

would `^neurodamus-core+common` work? would be nice but neurodamus-core is the end package